opus/import/obs_bundle_uranus_occs_earthbased.py

```
@@ -133,7 +133,57 @@ def _target_name(self):
     ### OVERRIDE FROM ObsProfile ###
     ################################
 
-    # Occultation type is set as 'STE' in obs_profile_pds4
+    def field_obs_profile_occ_type(self):
+        occ_type = self._index_col('rings:occultation_type')
+        if occ_type == 'stellar':
+            return self._create_mult('STE')
+        self._log_nonrepeating_error(
+            f'Unknown rings:occultation:type "{occ_type}"')
```

Ruff (0.15.10)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)